src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -277,7 +276,6 @@
 
 (defn view
   []
-  (rn/use-mount #(rf/dispatch [:centralized-metrics/check-modal metrics-modal/view]))
```

Comment:
For existing users who have not confirmed their choice, we will likely show the usage data screen after login. (Needs to discuss, cc @ilmotta)